### PR TITLE
Update nuget pipeline to use CentOS6

### PR DESCRIPTION
**Pull request by snnn**
**Description**: 

Update nuget pipeline to use CentOS6

**Motivation and Context**
- Why is this change required? What problem does it solve?
To get better compatibility with .Net Core.

- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

File: tools/ci_build/github/azure-pipelines/win-gpu-tensorrt-ci-pipeline.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
       displayName: 'Download test data and generate cmake config'
```
Comment (yuslepukhin) [inline tools/ci_build/github/azure-pipelines/win-gpu-tensorrt-ci-pipeline.yml]: >Download test data and generate cmake config' [](start = 20, length = 45)

Which part does the download? On my dev box I usualy have to do it manually

**Comment by yuslepukhin** [comment]
> - job: Windows_CI_GPU_Dev

Should TensorRT be reflected in the name?

---
Refers to: tools/ci_build/github/azure-pipelines/win-gpu-tensorrt-ci-pipeline.yml:2 in df5cd97. [](commit_id = df5cd97ac410d5e80285f871aa6fd55dc1736e65, deletion_comment = False)

File: tools/ci_build/github/azure-pipelines/win-gpu-tensorrt-ci-pipeline.yml (hunk 1 above)
Comment (snnn) [reply tools/ci_build/github/azure-pipelines/win-gpu-tensorrt-ci-pipeline.yml]: No, It didn't download the test data.

Comment (yuslepukhin) [reply tools/ci_build/github/azure-pipelines/win-gpu-tensorrt-ci-pipeline.yml]: Should the name of the task be changed then? It does mention Download test data

---
In reply to: [337250310](https://github.com/microsoft/onnxruntime/pull/2211#discussion_r337250310) [](ancestors = 337250310)

File: tools/ci_build/github/azure-pipelines/nuget/templates/gpu.yml [hunk 2]
```diff
@@ -27,23 +27,38 @@ jobs:
...
-    NuPackScript: |
-     set -e -x
-     mkdir $(Build.BinariesDirectory)/linux-x64
-     cp $(Build.BinariesDirectory)/Release/libonnxruntime.so $(Build.BinariesDirectory)/linux-x64
-     cd $(Build.BinariesDirectory)
-     zip -r linux-x64.zip linux-x64
-     cp $(Build.BinariesDirectory)/linux*.zip $(Build.ArtifactStagingDirectory)
-     ls -al $(Build.ArtifactStagingDirectory)
-
+- job: 'Linux_CI_GPU_Dev'
```
Comment (yuslepukhin) [inline tools/ci_build/github/azure-pipelines/nuget/templates/gpu.yml]: >Linux_CI_GPU_Dev [](start = 8, length = 16)

Should we say CentOS6_CI_GPU_Dev

File: tools/ci_build/github/azure-pipelines/win-gpu-tensorrt-ci-pipeline.yml (hunk 1 above)
Comment (snnn) [reply tools/ci_build/github/azure-pipelines/win-gpu-tensorrt-ci-pipeline.yml]: I'll submit a PR to fix it.

**Pull request merged**